config/routes.rb

```diff
@@ -1,8 +1,7 @@
 Rails.application.routes.draw do
+  
+  resources :comments
+  resources :messages
   root to: 'home#show'
 
-  if Rails.env.development?
-    mount LetterOpenerWeb::Engine, at: "/letter_opener"
```

Comment:
You can probably just remove the letter opener gem too. Don't think we'll have a lot of emails on this thing